tests/unit/Entity/PropertyTest.php

```diff
@@ -7,32 +7,20 @@
 use Wikibase\DataModel\Entity\PropertyId;
 use Wikibase\DataModel\Snak\PropertyNoValueSnak;
 use Wikibase\DataModel\Statement\StatementList;
+use Wikibase\DataModel\Term\AliasGroup;
+use Wikibase\DataModel\Term\AliasGroupList;
 use Wikibase\DataModel\Term\Fingerprint;
+use Wikibase\DataModel\Term\Term;
+use Wikibase\DataModel\Term\TermList;
 
 /**
  * @covers Wikibase\DataModel\Entity\Property
- * @covers Wikibase\DataModel\Entity\Entity
- *
- * @group Wikibase
- * @group WikibaseProperty
- * @group WikibaseDataModel
```

Comment:
This is unrelated to all tree commits in this chain. Please revert this and upload a separate patch. Personally, I prefer to have the said `@group WikibaseDataModel` in all tests.